bluesky/callbacks/broker.py

```diff
@@ -301,3 +301,37 @@ def stop(self, doc):
         self._start = None
         self.filenames = []
         super().stop(doc)
+
+
+class Exporter(CallbackBase):
+    def __init__(self, new_db, old_db, new_root=None, copy_kwargs=None):
+        """
+
+        Parameters
+        ----------
+        new_db: databroker.Broker
+        old_db: databroker.Broker
+        new_root: str
+        copy_kwargs: dict
+        """
+        self.new_db = new_db
+        self.old_db = old_db
+        self.copy_kwargs = copy_kwargs
+        self.new_root = new_root
+
+        self.start_uid = None
+
+    def start(self, doc):
+        # This is run on start just in case we miss a stop
```

Comment:
By what mechanism are we expecting to miss a stop?

Reply:
I think I thought that halt wouldn't put out a stop. I can remove it though.

Reply:
That is correct, but I wouldn't both designing around that case.